stock-filters/ChirpVillage/Builder.py

```diff
@@ -49,7 +52,7 @@ def construct_pointed_roof(level, coords, biome, height):
 def construct_pillars(level, coords, biome, height):
     minx, minz, miny, maxx, maxz = get_coords(coords)
     pillar_block = BlockUtils.get_beam_block(biome)
-    for y in range(miny, height):
+    for y in range(0, height):
```

Comment:
Is this to make the pillars go down to look like its supporting the building?

Reply:
Yep, I figured I'd make it go to the bottom to be sure that we could handle any height

Reply:
Rather then doing all they way to bedrock, you can go down 1 block, then check if that block is in surface_blocks. If it is in surface_blocks, replace with a piller, if not you have hit the bottom

Reply:
miny is the surface level, anything below that wont be in the surface. This is in case one corner has a block two or more levels below it. I put it to bedrock in case someone builds on a large cliff

Reply:
Does this mean the pillers will go all the way to bedrock if there is a block between he building and the bedrock? If so my proposed solution would fix that issue

Reply:
It means they always go to bedrock